cmd/envtool/main.go

```diff
@@ -96,9 +96,25 @@ func waitForPort(ctx context.Context, port uint16) error {
 			return nil
 		}
 
-		sleepCtx, sleepCancel := context.WithTimeout(ctx, time.Second)
-		<-sleepCtx.Done()
-		sleepCancel()
+		time.Sleep(time.Second)
+	}
+
+	return fmt.Errorf("failed to connect to 127.0.0.1:%d", port)
+}
+
+func waitForPostgresPort(ctx context.Context, port uint16) error {
+	logger := zap.S().Named("postgres.wait")
+
+	for ctx.Err() == nil {
+		var pgPool *pgdb.Pool
+		pgPool, err := pgdb.NewPool(fmt.Sprintf("postgres://postgres@127.0.0.1:%d/ferretdb", port), logger.Desugar(), false)
```

Comment:
can the hard-code be moved to config / flags somehow?

Reply:
Yes, when our main branch is not broken